repose-aggregator/components/filters/regex-rbac-filter/src/main/resources/META-INF/schema/config/regex-rbac.xsd

```diff
@@ -0,0 +1,145 @@
+            </xs:documentation>
+        </xs:annotation>
+
+        <xs:sequence>
+            <xs:element name="delegating" type="DelegatingType" minOccurs="0" maxOccurs="1"/>
+            <xs:element name="resources" type="ResourcesType" minOccurs="1" maxOccurs="1"/>
+        </xs:sequence>
+
+        <xs:attribute name="mask-rax-roles-403" type="xs:boolean" use="optional" default="false">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>
+                        Mask rax-roles with NOT FOUND (404) and METHOD NOT ALLOWED (405) errors.
+                        By default rax-roles responds with a FORBIDDEN (403) if there is a role mismatch,
+                        if this is set to true, then the response will be NOT FOUND (404) if no methods are
+                        accessible or METHOD NOT ALLOWED (405) if some methods are available.
+                    </html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+    </xs:complexType>
+
+    <xs:complexType name="DelegatingType">
+        <xs:annotation>
+            <xs:documentation>
+                <html:p>Whether or not you would like this filter to populate the delegation headers. Inclusion means you do.</html:p>
+            </xs:documentation>
+        </xs:annotation>
+        <xs:attribute name="quality" type="QualityType" use="optional" default="0.3">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>
+                        What quality you want any output headers to be.
+                        When setting up a chain of delegating filters the highest quality number will be the one that is eventually output.
+                        Default is 0.3
+                    </html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+        <xs:attribute name="component-name" type="xs:string" use="optional" default="regex-rbac">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>
+                        Sets the delegation component name for this validator.
+                    </html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+    </xs:complexType>
+
+    <xs:simpleType name="QualityType">
+        <xs:restriction base="xs:double">
+            <xs:minInclusive value="0"/>
+            <xs:maxInclusive value="1.0"/>
+        </xs:restriction>
+    </xs:simpleType>
+
+    <xs:complexType name="ResourcesType">
+        <xs:annotation>
+            <xs:documentation>
+                <html:p>
+                    This is a list of resources, HTTP methods, and roles allowed to access them.
+
+                    The special HTTP methods ANY and ALL are shorthand for the four standard REST methods
+                    (GET, PUT, POST, DELETE). Other methods are supported, but they must be explicitly stated.
```

Comment:
Patch is not included in any/all?

Reply:
It wasn't for the Simple RBAC.

Reply:
I can add it here if we will need it.

Reply:
That's super non-intuitive. why did we do this the first time?

Reply:
The original was intended to solve "simple" issues and worked under the assumption of only supporting basic CRUD operations.

Reply:
I can make `ANY`/`ALL` be truly any or all and basically just not check the method.

Reply:
I'd say go ahead and support them all on any/all. lets not make them weirdly counter intuitive